src/api/form/PDFTextField.ts

```diff
@@ -669,6 +669,7 @@ export default class PDFTextField extends PDFField {
       borderColor: options.borderColor,
       borderWidth: options.borderWidth ?? 0,
       rotate: options.rotate ?? degrees(0),
+      hidden: options.hidden,
```

Comment:
Would you please add this to all the different field types? (`PDFButton`, `PDFCheckBox`, `PDFDropdown`, `PDFForm`, `PDFOptionList`, `PDFRadioGroup`, `PDFTextField`)

Reply:
252b71b